includes/utilities.php

```diff
@@ -47,8 +47,11 @@ function sanitise_and_normalise_url( $unsafe_url ) {
 		$unsafe_url = add_leading_slash( $unsafe_url );
 	}
 
+	// remove any parameters and trailing slash from the url.
```

Comment:
Very minor, consistency in comment style
```suggestion
	// Remove any parameters and trailing slash from the url.
```